### PR TITLE
remove malware (mac addition needed)

### PR DESCRIPTION
**Pull request by randomairborne**
antivirus for win only is better then antivirus for none at all

### CODE REVIEW

**Review by TechnoShip123 — DISMISSED** [review]
Content is accurate

File: _help/removing-malware/index.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title:  "ADWcleaner and Malwarebytes"
+name: "remove-malware"
+desc: "Running ADWcleaner and Malwarebytes"
+---
+
+# Running an antivirus/antimalware scan on your computer
+
+Some malware can interfere with minecraft and its RAM allocation, causing crashes and login issues.  Malwarebytes ADWcleaner needs to be used as it is known to be safe, effective, and not interfere with minecraft
```
Comment (LordRalex) [inline _help/removing-malware/index.markdown:10]: Minecraft. And RAM allocation doesn't matter in most cases. That's only specific to 1 program, which isn't "malware".
I don't like how the second sentence is worded here either.

File: _help/removing-malware/index.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title:  "ADWcleaner and Malwarebytes"
```
Comment (LordRalex) [inline _help/removing-malware/index.markdown:3]: The tool is called AdwCleaner. Please fix this across the board.

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: _help/removing-malware/index.markdown (hunk 1 above)
Comment (randomairborne) [reply _help/removing-malware/index.markdown:10]: I dont truly understand why it interferes with minecraft, this is what I understood from what I could.  enlightenment is appreciated

File: _help/removing-malware/index.markdown (hunk 2 above)
Comment (randomairborne) [reply _help/removing-malware/index.markdown]: Done.

**Review by LordRalex — APPROVED** [review]

**Pull request merged**